### PR TITLE
User agent summary update and close clean up

### LINKED ISSUE #604: User agent update
The `Custom user agent` section of the settings page currently has the following description `For exchange rate fetching`.

image

it should be changed to `For HTTP request`.

Also, it is not showing any user agent being used. However, it is expected to show the default user agent if there are no custom values.

### LINKED ISSUE #647: Replace all instances of "seed phrase", "spending password" and "startup password" with 'seed words', 'spending passphrase', and 'startup passphrase'
Find/Replace ALL instances of: "seed phrase" with 'seed words', 
Find/Replace All:  "spending password" with 'spending passphrase', and 
Find/Replace All: "startup password" with 'startup passphrase' @Sirmorrison @devchoplife

### PR DESCRIPTION
**Pull request by Sirmorrison**
fix #604
fix #647
- update user agent summary message
- change passphrase text to password
- removes the security page info modal
- hides the auto purchase toggle button
- -remove redundant util file and functions.

### CODE REVIEW

File: ui/page/settings_page.go [hunk 1]
```diff
@@ -490,12 +498,19 @@ func (pg *SettingsPage) Handle() {
 				NegativeButton(values.String(values.StrCancel), func() {}).
 				PositiveButton(values.String(values.StrConfirm), func(password string, pm *modal.PasswordModal) bool {
 					go func() {
+						var error string
 						err := pg.wal.GetMultiWallet().RemoveStartupPassphrase([]byte(password))
 						if err != nil {
-							pm.SetError(err.Error())
+							if err.Error() == "invalid_passphrase" {
+								error = "invalid password"
+							} else {
```
Comment (beansgum) [inline ui/page/settings_page.go]:
```suggestion
							if err.Error() == dcrlibwallet.ErrInvalidPassphrase {
								error = "Invalid password"
							} else {
```

File: ui/page/main_page.go [hunk 2]
```diff
@@ -264,8 +264,8 @@ func (mp *MainPage) UnlockWalletForSyncing(wal *dcrlibwallet.Wallet) {
 				err := mp.WL.MultiWallet.UnlockWallet(wal.ID, []byte(password))
 				if err != nil {
 					errText := err.Error()
-					if err.Error() == "invalid_passphrase" {
-						errText = "Invalid passphrase"
+					if err.Error() == "invalid password" {
```
Comment (beansgum) [inline ui/page/main_page.go]:
```suggestion
					if err.Error() == dcrlibwallet.ErrInvalidPassphrase {
```

File: ui/page/settings_page.go [hunk 3]
```diff
@@ -479,6 +486,7 @@ func (pg *SettingsPage) Handle() {
 							m.SetLoading(false)
 							return
 						}
+						pg.Toast.Notify("Startup password activated")
```
Comment (beansgum) [inline ui/page/settings_page.go]:
```suggestion
						pg.Toast.Notify("Startup password enabled")
```

File: ui/page/settings_page.go [hunk 4]
```diff
@@ -490,12 +498,19 @@ func (pg *SettingsPage) Handle() {
...
+							if err.Error() == "invalid_passphrase" {
+								error = "invalid password"
+							} else {
+								error = err.Error()
+							}
+							pm.SetError(error)
 							pm.SetLoading(false)
 							return
 						}
+						pg.Toast.Notify("Startup password deactivated")
```
Comment (beansgum) [inline ui/page/settings_page.go]:
```suggestion
						pg.Toast.Notify("Startup password disabled")
```

**Review by beansgum — DISMISSED** [review]

File: ui/page/settings_page.go [hunk 5]
```diff
@@ -272,7 +272,7 @@ func (pg *SettingsPage) agent() layout.Widget {
 							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 								layout.Rigid(pg.subSectionLabel(values.String(values.StrCustomUserAgent))),
 								layout.Rigid(func(gtx C) D {
-									txt := pg.Theme.Body2(values.String(values.StrUserAgentSummary))
+									txt := pg.Theme.Body2(values.String("For HTTP request"))
```
Comment (dreacot) [inline ui/page/settings_page.go]:
```suggestion
									txt := pg.Theme.Body2("For HTTP request")
```

**Review by dreacot — CHANGES_REQUESTED** [review]

**Comment by dreacot** [comment]
currently a default user agent isn't shown if there is no custom agent set

**Comment by Sirmorrison** [comment]
> currently a default user agent isn't shown if there is no custom agent set

Yeah at the moment, there is no default user agent from the config settings.

**Review by dreacot — CHANGES_REQUESTED** [review]
This says enter your seed phrase

Screenshot from 2021-09-29 19-38-12

**Review by dreacot — DISMISSED** [review]

**Review by dreacot — APPROVED** [review]

**Pull request merged**